lib/sles4sap/sap_deployment_automation_framework/basetest.pm

```diff
@@ -79,11 +79,12 @@ sub full_cleanup {
 
     # Trigger SDAF remover script to destroy 'workload zone' and 'sap systems' resources
     # Clean up all config files, keys, etc.. on deployer VM
+    my %cleanup_results;
     if ($redirection_works) {
         load_os_env_variables();
         az_login();
         _sdaf_ibsm_teardown() if get_var('IS_MAINTENANCE');
-        sdaf_cleanup();
+        %cleanup_results = %{sdaf_cleanup()};
```

Comment:
This stores the cleanup results and won't block other important cleanup tasks to continue. 
Failures are reported later.